test/integration/processing/settlement/get-last-settlement.test.js

```diff
@@ -0,0 +1,101 @@
+const db = require('../../../../app/data')
+
+const { getLastSettlement } = require('../../../../app/processing/settlement')
+const { SETTLEMENT } = require('../../../mock-components/mock-dates')
+let currentSettlement
+let previousSettlement
+
+describe('process settlement', () => {
+  beforeAll(async () => {
+    await db.sequelize.truncate({
+      cascade: true,
+      restartIdentity: true
+    })
+  })
+
+  beforeEach(async () => {
+    currentSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+    currentSettlement.settlementDate = new Date(SETTLEMENT.DATE)
+    previousSettlement = JSON.parse(JSON.stringify(require('../../../mock-settlement')))
+    previousSettlement.settlementDate = new Date(2022, 1, 7)
```

Comment:
Can we make this date reference `currentSettlement` and take 1 day away?

Reply:
Can see you have done this elsewhere:

`previousSettlement.settlementDate = moment(currentSettlement.settlementDate).subtract(1, 'day')`

Reply:
Orrrrr maybe you updated this exact line with that 😆